LocationManager/INTULocationManager/INTULocationManager.m

```diff
@@ -503,7 +503,7 @@ - (void)processLocationRequests
     CLLocation *mostRecentLocation = self.currentLocation;
     
     for (INTULocationRequest *locationRequest in self.locationRequests) {
-        if (locationRequest.hasTimedOut) {
+        if (locationRequest.isRecurring == NO && locationRequest.hasTimedOut) {
```

Comment:
This is a good change to be explicit, but because subscriptions cannot time out (and never have a timeout value set), `hasTimedOut` will never return YES for them (so this change shouldn't actually end up changing any behavior).